generators/app/index.js

```diff
@@ -156,9 +156,15 @@ module.exports = class extends BaseGenerator {
             type: String
         });
 
-        // This adds support for a `--blueprint` flag which can be used to specify a blueprint to use for generation
+        // NOTE: Deprecated!!! Use --blueprints instead
         this.option('blueprint', {
-            desc: 'Specify a generator blueprint to use for the sub generators',
+            desc: 'Specify one or more generator blueprints to use for the sub generators',
```

Comment:
It could actually be used as  `--blueprint A,B` similar to `--blueprints`. But you are right, I have added `DEPRECATED:` and restored the old description